### DISCUSSION TITLE
Room assistant setup with iPhone

### DISCUSSION THREAD

**Discussion by mikepflu** [mKeRix/room-assistant#354] [Q&A]
Hi, I've been trying off and on for a week or two to get room assistant setup to track room presence using apple devices, mainly iPhones.  Granted, I usually only spend a few minutes here and there, but I'm finally trying to put some real time towards it.

I saw in another post recently the mention of the companion app.  Does that help with room presence?

I think I'm just at a bit of a loss as to how to configure it completely.  I currently have one Raspberry Pi 3 setup with room assistant.

This is my local.yml for room-assistant:

`
global:
  instanceName: Living Room
  integrations:
    - homeAssistant
    - bluetoothClassic
homeAssistant:
  mqttUrl: mqtt://192.168.50.20:1883
bluetoothClassic:
  minRssi: -20
  addresses:
    # Mike iPhone
    - '40:9c:xx:xx:xx:xx'
    # Lisa iPad
    - 'e0:b5:xx:xx:xx:xx'
`

This is the block I have in my configuration.yaml for home assistant:
`
device_tracker:
- platform: mqtt
  devices:
    mike_iphone: 'room-assistant/device_tracker/bluetooth-classic-40-9c-xx-xx-xx-xx-tracker/state'
    lisa_ipad: 'room-assistant/device_tracker/bluetooth-classic-e0-b5-xx-xx-xx-xx-tracker/state'
  payload_home: 'true'
  payload_not_home: 'false'
  source_type: bluetooth
`

Using mqtt.fx on my mac and looking at mqtt, I do see a number of entries under room-assistant/, including one that matches with my iphone (mike_iphone) but `room-assistant/device_tracker/bluetooth-classic-40-9c-xx-xx-xx-xx-tracker/state` always says false, and `room-assistant/sensor/bluetooth-classic-40-9c-xx-xx-xx-xx/state` always says not home.

Currently, I'm sitting in my living room with my iPhone right next to me and the Raspberry Pi is about 10 feet across the room in the entertainment center.

Am I grabbing the wrong bluetooth mac?  I'm going to iPhone > Settings > General > About > Bluetooth, and this is the address that I put in the room-assistant config and configuration.yaml.

Once I get this working, I have a pi zero that I would like to add to the mix.

Thanks for any help!

**Comment by mKeRix** [comment]
To me it sounds like you are using the correct MAC address. Is there anything else running on the Pi that might use Bluetooth? Do you see anything notable in the room-assistant logs? And what do you see when you go to `http://<room-assistant-ip>:6415/entities` in your browser? That should be a page listing the internal entity registry along with some more details for the measured RSSIs and so on.

**Reply by mikepflu** [reply]
There shouldn't be anything else running on the Pi using the bluetooth.  I don't really see anything in the logs either.  I didn't know about the URL for checking on entities, but that's good to know.  There's nothing notable in there either.  At this time, I'm trying to setup the companion app with BLE and I'm not having much success there either.
